ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -283,6 +283,33 @@
                 width: 200
             }];
             return fields;
+        },
+        onRefresh: function() {
+            var me = this, ctrl = window.globalApp.getController("DashboardController"),
+            id = ctrl.getCurrentDashboard().id.split("/")[ctrl.getCurrentDashboard().id.split("/").length -1],
+            portletId = me.ownerCt.id;
```

Comment:
Please reformat this with one variable per line and/or indentation, e.g.
```
var me = this,
    ctrl = window.globalApp.getController("DashboardController"),
    id = ctrl.getCurrentDashboard().id.split("/")[ctrl.getCurrentDashboard().id.split("/").length -1],
    portletId = me.ownerCt.id;
```
Also, nobody will be upset if you captured the result of `ctrl.getCurrentDashboard().id.split("/")` into a variable first and used it rather having to repeat the expression twice, e.g.
```
var me = this,
    ctrl = window.globalApp.getController("DashboardController"),
    idAsArray = ctrl.getCurrentDashboard().id.split("/"),
    id = idAsArray[idAsArray.length -1],
    portletId = me.ownerCt.id;
```